packages/soy-compiler/lib/pipelines/compileSoy.js

```diff
@@ -294,7 +294,7 @@ function replaceTemplateRequires() {
 		let contents = file.contents.toString();
 
 		const importMap = {};
-		const requireRegex = /var\s+\$import(\d*)\s*=\s*goog\.require\('(\w+)\.incrementaldom'\);/g;
+		const requireRegex = /var\s+\$import(\d *)\s *=\s*goog\.require\('(\w(.*))+\.incrementaldom'\);/g;
```

Comment:
Why the spaces around the `(\d *)`? Also, `(.*)` is matching any character, not just dots. we should be able to get rid of `\w` at that point, shouldn't we?

Reply:
These spaces should not be there 😂 
You are sure, I'm not escaping `.`. 

I'll fix it.